src/content/docs/synthetics/synthetic-monitoring/private-locations/troubleshoot-private-locations.mdx

```diff
@@ -113,3 +113,11 @@ The private minion must have access to `https://synthetics-horde.nr-data.net` in
 **Network IOPS:**
 
 Network IOPS can also be a good place to troubleshoot if using a cloud provider for your CPM. Both Docker and Kubernetes CPMs require a decent amount of write throughput. Analyze your average IOPS and determine the appropriate volume size needed for your disk operations per second.
+
+## Port conflicts running Docker containers on the host [#port-conflicts-docker]
+
+If you're running more than one Docker private location container in the same host, you'll have port conflicts. To avoid this port contention, you'd make sure you do the following when you start setting up job managers:
```

Comment:
@nbaenam I'd also suggest you drop the "would" verb form here: To avoid this port contention, make sure you do the following when you start setting up job managers: